wolfram.el

```diff
@@ -138,16 +138,24 @@
     (insert (format "# \"%s\" (searching)\n"
                     (propertize query 'face 'wolfram-query)))))
 
-(defun wolfram--append-pods-to-buffer (buffer pods)
-  "Appends all of the pods to a specific buffer."
+(defun wolfram--delete-in-progress-notification ()
+  "Switch to WolframAlpha buffer and delete the \"(searching)\" notification.
+That notification indicates that the search is still in progress. This function
+removes that notification."
+  (wolfram--switch-to-wolfram-buffer)
+  (goto-char (point-max))
+  (search-backward " (searching)")
+  (let ((inhibit-read-only t))
+    (replace-match ""))
+  (goto-char (point-max)))
+
+(defun wolfram--append-pods-to-buffer (pods)
+  "Appends all pods from PODS to WolframAlpha buffer."
+  (wolfram--delete-in-progress-notification)
   (let ((inhibit-read-only t))
-    (goto-char (point-max))
-    (search-backward " (searching)")
-    (replace-match "")
-    (goto-char (point-max))
     (dolist (pod pods)
-      (wolfram--append-pod pod))
-    (insert "\n")))
+      (wolfram--append-pod pod)
+      (insert "\n"))))
```

Comment:
Nice catch! It was an overlook on my part. I did not intend the headings to be separated by 2 newlines. I have now fixed it.